packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue

```diff
@@ -68,6 +91,9 @@ const emit = defineEmits(['update:selection']);
 }
 
 .dropdown-option-group {
+	display: flex;
+	gap: 0.2rem;
+	align-items: center;
 	font-size: var(--font-caption);
 	color: var(--gray-600);
 }
```

Comment:
I think it'd be nice for the label and date to have space in between
```suggestion
   justify-content: space-between;
}
```